LaTeX/Parts/Chapter4.tex

```diff
@@ -38,6 +41,13 @@ \section{Broker}
 	</broker>
 </broker>
 \end{lstlisting}
+The broker is the central component whose resilience is tested by the Scenario.
+The user needs to define:
```

Comment:
Maybe start the sentence with 'Herefore', could be more fluently readable.

Reply:
'For this' is better, but good idea.